play-ws-standalone-json/src/main/scala/play/api/libs/ws/JsonBodyReadables.scala

```diff
@@ -18,7 +21,20 @@ trait JsonBodyReadables {
    * }}}
    */
   implicit val readableAsJson: BodyReadable[JsValue] = BodyReadable { response =>
-    Json.parse(response.body)
+    val body = response.bodyAsBytes
+    Json.parse(body.decodeString(detectEncoding(body)))
```

Comment:
Why not use `Json.parse(response.body.toArray[Byte])`? It should detect the encoding for you.